fiat-roles/src/main/java/com/netflix/spinnaker/fiat/providers/ApplicationResourcePermissionSource.java

```diff
@@ -27,12 +27,12 @@
 import java.util.Map;
 import javax.annotation.Nonnull;
 
-public final class Front50ApplicationResourcePermissionSource
+public final class ApplicationResourcePermissionSource
```

Comment:
nit: No disagreement here, but interested to know rationale.

Reply:
Essentially, this class contains a bit of logic around `EXECUTE` and `CREATE` permissions that we want to apply by default but these are not specific to processing front50 permissions - they could be any permission source.

Reply:
With that I should say, the impotence for this change is essentially so that we always process resource `EXECUTE` permissions last (since it falls back to `READ` if not set).